CONTRIBUTING.md

```diff
@@ -93,8 +93,12 @@ The FreeCAD Contribution Process is expressed here with the following specific g
 
 ## 8. Project Administration
 
-1. The project Administrators will manage the set of project Maintainers.  They SHALL maintain a sufficiently large pool of Maintainers to ensure their succession and permit timely review of contributions.
-2. Contributors who have a history of successful PRs and have demonstrated continued professionalism should be invited to be Maintainers.
-3. Administrators SHOULD remove Maintainers who are inactive for an extended period, or who repeatedly fail to apply this process accurately.
-4. The list of Maintainers SHALL be publicly accessible and reflective of current activity on the project.
-5. Administrators SHOULD block or ban “bad actors” who cause stress, animosity, or confusion to others in the project. This SHOULD be done after public discussion, with a chance for all parties to speak. A bad actor is someone who repeatedly ignores the rules and culture of the project, who is hostile or offensive, who impedes the productive exchange of information, and who is unable to self-correct their behavior when asked to do so by others.
+1. Project Administrators are those individuals who are members of the FreeCAD Github organization and have the role of 'owner'.  They have the task of administering the organization including adding and removing individuals from various teams.
+2. Project Administrator is a technical role necessitated by the GitHub platform. Except for the specific exceptions listed below, the Project Administrators do not make the decision about individual team members. Rather, they carry out the collective wishes of the Maintainers team. Project Administrators will be selected from the Maintainers team but the Maintainers themselves.
+3. To ensure continuity there SHALL be at least four Project Administrators at all times.
+4. The project Administrators will manage the set of project Maintainers.  They SHALL maintain a sufficiently large pool of Maintainers to ensure their succession and permit timely review of contributions. If the pool of Maintainers is insufficient, the Project Administrators will request that the Maintainers select additional individuals to add.
```

Comment:
Currently it's only the maintainers who select new maintainers. Should there be a proposal or even discussion phase in the forum before doing so?

Reply:
These changes are basically deliberately leaving out how Maintainers as a group choose to make decisions. The point here is simply that it is *not* "Administrators"/"owners", etc. making that call: Maintainers are the final authority. How we actually exercise that is left as an "implementation detail". If the group of Maintainers wants to codify some process or rules, they/we are clearly free to do so, and the "administrators" have no special say in the matter.